sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -149,6 +149,11 @@ def get_api_version(kwargs):
         raise ValueError("Unsupported API version '{}'. Please select from:\n{}".format(api_version, versions))
     return api_version or _SUPPORTED_API_VERSIONS[-1]
 
+def get_version_id(self_vid, kwargs):
+    # type: (str, Dict[str, Any]) -> str
```

Comment:
I think the type here would actaully be `(Optional[str], Dict[str, Any]) -> Optional[str]` as `self_vid` can be `None`.